index.html

```diff
@@ -7,6 +7,130 @@
     <title>shop-cart</title>
     <script src="index.js"></script>
     <link rel="stylesheet" href="style.css" />
+    <script src="https://kit.fontawesome.com/bf364dafc4.js" crossorigin="anonymous"></script>
   </head>
-  <body></body>
+  <body>
+    <header>
+      <a class="logo" href="/" title="logo" aria-label="link to homepage">Shop-Cart</a>
+      <div class="wrapper__Navbar">
+        <div class="wrapper__Links">
+          <a
+            class="Inter"
+            title="Support"
+            aria-label="link to support page"
+            style="margin-right: 2vw"
+            href="#"
+            >Support
+          </a>
+          <div class="separator" style="min-width: 3vw"></div>
+          <a class="Inter" href="#" title="Services" aria-label="link to services page">
+            Services</a
+          >
+        </div>
+        <div class="separator" style="min-width: 5vw"></div>
+        <button
+          class="dropdown__Account"
+          aria-label="open account menu"
+          type="menu"
+          style="margin: 0; padding: 0"
+          onclick="test(e)"
+        >
+          <a href="#" title="Account menu" aria-label="account dropdown menu">Account </a>

```

Comment:
`<a>`inside button the same mistake as in HTML task 😉

Reply:
Well noted.